data/commands/help.json

```diff
@@ -22,7 +23,7 @@
         "Account Management & Information",
         "",
         "You are able to change your own password and nickname by using the \"/account\" command. In",
-        "addition, VIPs have the ability to create aliases for their account through this command as well.",
+        "addition, VIPs have the ability to create aliases (max is 2 aliases) for their account through this command as well.",
```

Comment:
"addition, VIPs can create up to two aliases for their account through this command as well.",